### DISCUSSION TITLE
[BP#57] Utiliser tous les niveaux de cache du CMS

### DISCUSSION THREAD

**Discussion by devcrafting** [cnumr/best-practices#209] [Bonnes pratiques] [answered]
Spécifique aux CMS? On pourrait élargir , d'ailleurs c'est peut-être le cas déjà via différentes BP autour de la mise en cache?

Titre autour des CMS et contenu très spécifique Drupal...que je ne connais pas donc je ne peux pas trop juger de la pertinence et si c'est à jour, mais ça me gène que cette BP soit spécifique à Drupal...

**Comment by ACTLEM** [comment]
Drupal est utilisé comme exemple mais idem je ne connais pas assez Drupal pour savoir si c'est toujours d'actualité. @yannkozon un avis ?
Sinon, cela me semble opportun de garder cette BP car les CMS restent très majoritaires sur le web.

**Comment by Fred-artwai** [comment]
Alors si je puis me permettre, WordPress étant le CMS le plus répandus, nous devrions remplacer ou compléter toutes les références à Drupal par WordPress. Ici, on peut reprendre les plugins Total Cache, ou WP Rocket comme exemple avec WordPress.

**Comment by ACTLEM** [comment, marked as answer]
J'ai créé la BP pour modification (suppression référence à un CMS en particulier dans la description + exemple Wordpress plus que Drupal): https://github.com/cnumr/best-practices/issues/264
